docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -0,0 +1,61 @@
+# Forecast quarterly house sales using MindsDB
+
+## Introduction
+
+In this short example we will produce forecasts for a multivariate time series.
+
+## Data and setup
+
+The dataset we will use is the pre-processed version of the "House Property Sales" [Kaggle competition](https://www.kaggle.com/datasets/htagholdings/property-sales), in particular, the "ma_lga_12345.csv" file, which tracks quarterly moving averages of house sales aggregated by type and amount of bedrooms in each listing.
+
+Make sure you have access to a working MindsDB installation (either local or via cloud.mindsdb.com), and either load it into a table in your database of choice or upload the file directly to the special `FILES` datasource (via SQL or GUI). Once you've done this, proceed to the next step. For the rest of the tutorial we'll assume you've opted for the latter option:
+
+```
+SHOW TABLES FROM files;
+
+SELECT * FROM files.HR_MA LIMIT 10;
```

Comment:
Maybe we can mention that HR_MA is the name of the uploaded dataset?

Reply:
Oh, good point yes, I will mention it!